doc/release/README.launcher.rst

```diff
@@ -0,0 +1,228 @@
+
+When compiling Chapel programs for multiple locales, a launcher binary
+is typically created that will execute the appropriate command(s) to
+get your program started. For example, when compiling for multiple
+locales, typically two binaries will be generated by the compiler
+(e.g., a.out and a.out_real). The first binary contains code to get
+your program up and running on multiple locales while the second
+contains your actual program code.
+
+The goals of the launcher binary are: 
+
+#. to wrap details of job startup in a portable way so that new users
+   can quickly get Chapel programs up and running on an unfamiliar
+   platform.
+
+#. to perform command-line parsing and error checking prior to
+   waiting in a queue or firing off a parallel job in order to save
+   time and resources related to simple errors/typos in the command
+   line.
+
+#. to preserve Chapel's global-view programming model by permitting
+   the user to run their program using a single binary (corresponding
+   to the single logical task that executes main()) without getting
+   bogged down in questions of numbers of nodes, numbers of cores per
+   node, numbers of program instances to start up, etc.
+
+#. if necessary, to coordinate runtime functional activity, such as
+   I/O.
+
+Executing a Chapel program using the verbose (``-v``) flag will typically
+print out the command(s) used to launch the program.
+
+Executing using the help (``-h``/``--help``) flag will typically print out
+any launcher-specific options in addition to the normal help message for
+the program itself.
+
+Currently supported launchers include:
+
+===================  ====================================================
+Launcher Name        Description
+===================  ====================================================
+amudprun             GASNet launcher for programs running over UDP        
+aprun                Cray application launcher using aprun                
+gasnetrun_ibv        GASNet launcher for programs running over Infiniband 
+gasnetrun_mpi        GASNet launcher for programs using the MPI conduit   
+lsf-gasnetrun_ibv    GASNet launcher using LSF (bsub) over Infiniband     
+pbs-aprun            Cray application launcher using PBS (qsub) + aprun   
+pbs-gasnetrun_ibv    GASNet launcher using PBS (qsub) over Infiniband     
+slurm-gasnetrun_ibv  GASNet launcher using SLURM over Infiniband          
+slurm-srun           native SLURM launcher                                
+none                 do not use a launcher                                
+===================  ====================================================
+
+A specific launcher can be explicitly requested by setting the
+``CHPL_LAUNCHER`` environment variable. If left unset, a default is picked as
+follows:
+
+
+* if ``CHPL_PLATFORM`` is cray-xc, cray-xe, cray-xk, or cray-xt:
+
+  ==================================  ===================
+  If                                  Then CHPL_LAUNCHER=
```

Comment:
I think it looks a little cleaner if you remove the "Then " and "=" from "Then Chpl_LAUNCHER=", same for all 3 tables in this file